### ISSUE TITLE
Terra app doesn't work on IE 11

### ISSUE THREAD

**Issue by bansalrachita** [cerner/terra-core#3454]
# Bug Report

## Description
Building a Terra application with create-react-app gives an error on IE but works on other browsers.

![image](https://user-images.githubusercontent.com/10090750/121579048-312fb700-c9e0-11eb-8f7f-e62b56aef244.png)

## Steps to Reproduce
- Created a create-react-app application.
- Added translations by running `terra-aggregate-translations` script.
- Using Terra components `Base` and `ThemeProvider` from Terra as follows,
```
export const AppBase: React.FC<AppBaseProps> = ({
  children,
}: React.PropsWithChildren<AppBaseProps>) => {
  return (
    <Base locale="en">
      <ThemeProvider themeName="orion-fusion-theme">
        {children}
      </ThemeProvider>
    </Base>
  );
};
```
- Built the application using CRA.
- IE 11 returns a console.error and the page is blank

## Expected Behavior
This build seems to work on other browsers (viz. Chrome, Safari and Firefox) with their latest versions.

## Environment
* Browser Name and Version: IE 11 
* Node/npm Version: Node 10.x
* Webpack Version: CRA (version given below)
* Operating System and version (desktop or mobile): Windows 8.1 desktop
* Package.json:
```
"dependencies": {
    "@testing-library/jest-dom": "^5.11.4",
    "@testing-library/react": "^11.1.0",
    "@testing-library/user-event": "^12.1.10",
    "@types/node": "^15.6.1",
    "@types/react": "^17.0.9",
    "@types/react-dom": "^17.0.6",
    "@types/react-router-dom": "^5.1.7",
    "classnames": "^2.3.1",
    "node-sass": "^4.14.1",
    "react": "^17.0.2",
    "react-dom": "^17.0.2",
    "react-intl": "^2.9.0",
    "react-router-dom": "^5.2.0",
    "react-scripts": "4.0.3",
    "terra-aggregate-translations": "^1.0.0",
    "terra-application": "^1.41.1",
    "terra-avatar": "^3.12.0",
    "terra-badge": "^3.50.0",
    "terra-base": "^5.0.0",
    "terra-theme-provider": "^4.8.0",
    "typescript": "^4.3.2",
    "web-vitals": "^1.0.1"
  },
  "scripts": {
    "start": "react-scripts start",
    "build": "react-scripts build",
    "test": "react-scripts test",
    "eject": "react-scripts eject",
    "aggregate-translations": "tt-aggregate-translations -l ['en'] -d ./translations -o ./node_modules/terra-i18n/node_modules -f es6",
    "prestart": "yarn aggregate-translations"
  },
```

**Comment by mjhenkes** [comment]
@bansalrachita the biggest problem we run into with CRA applications that are trying to use terra is that we require a more complex webpack config. In this case I'm wondering if the aggregated translations aren't found because the aggregate translations folder isn't added as a resolve module: https://github.com/cerner/terra-toolkit/blob/main/packages/webpack-config-terra/src/webpack.config.js#L41

I'd also recommend that you use terra-application-base instead of terra-base.
https://engineering.cerner.com/terra-ui/application/terra-application/components/application-base

And if i might ask, what are you trying to build with terra?
